cfg/cppcheck-cfg.rng

```diff
@@ -116,7 +116,15 @@
             <element name="leak-ignore"><empty/></element>
           </optional>
           <optional>
-            <element name="use-retval"><empty/></element>
+            <element name="use-retval"><empty/>
+              <optional>
+                <attribute name="type">
+                  <data type="string">

```

Comment:
Would this work?

              <attribute name="type">
                <value>error-code</value>
              </attribute>